src/istio/utils/filter_names.cc

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2018 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "include/istio/utils/filter_names.h"
+
+namespace istio {
+namespace utils {
+
+// TODO: using more standard naming, e.g istio.jwt, istio.authn
+const char FilterName::kJwt[] = "jwt-auth";
+const char FilterName::kAuthentication[] = "istio_authn";
```

Comment:
why can't we fix the filter names now? better now than late. 
Also, why not std::string ?

Reply:
It requires accompany changes from Pilot, so I want to do it in a smaller PR with just that change.

Reply:
for std::string, just a personal style. This is just a const literal, and I don't see the need to use string for it.